lab-guides/01-AzureLocalCloudDeployment/readme.md

```diff
@@ -491,35 +494,40 @@ $LCMCredentials= New-Object System.Management.Automation.PSCredential ($LCMUserN
 
 ### Task05 - Create Azure Resources
 
-Following script will simply create Resource Group and ARC Gateway (optional).
+Following script will simply create Resource Group and Arc Gateway (optional).
 
 ```PowerShell
 $GatewayName="ALClus01-ArcGW"
 $ResourceGroupName="ALClus01-RG"
+
+#List of shortnames for Azure regions (Azure Shell)
+#az account list-locations -o table
 $Location="eastus"
 
-#login to azure
+#Login to azure
     #download Azure module
     Install-PackageProvider -Name NuGet -MinimumVersion 2.8.5.201 -Force
+    
+    #Starting with AzL 24H2 2507, az.accounts is not longer failing and will continue to work with the latest version of this module.
     if (!(Get-InstalledModule -Name az.accounts -ErrorAction Ignore)){
         Install-Module -Name Az.Accounts -Force 
```

Comment:
when a user has more access than one tenant we need to add comment for

Connect-AzAccount -Tenant 'TenantID' 

it will not ask but use the default tenant.